models/Comment.php

```diff
@@ -66,60 +62,63 @@ function get_comments(){
 
     // Fonction permettant de supprimer un commentaire
     function delete_comment(){
-        if (isset($_GET['delete']) AND !empty($_GET['delete'])) {
+        $delete = filter_input(INPUT_GET, 'delete', FILTER_SANITIZE_NUMBER_INT);
         
-        $delete = (int) $_GET['delete'];
-
-        $db = getPdo();
-        
-        $req = $db->prepare('DELETE FROM comments WHERE id = ?');
+        $req = $this->db->prepare('DELETE FROM comments WHERE id = ?');
         $req->execute(array($delete));
 
-        header("Location: index.php?page=dashboard");
-        
-        exit();    
-        }
+        header("Location: /dashboard");
     }
 
     // Fonction permettant d'approuver un commentaire
     function approve_comment(){
-        if (empty($_GET['approve']) || !ctype_digit($_GET['approve'])) {
-            die("Ho ! Fallait préciser le paramètre id en GET !");
-        }
-
-        $id = $_GET['approve'];
-
-        $db = getPdo();
+        $id = filter_input(INPUT_GET, 'approve', FILTER_SANITIZE_NUMBER_INT);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5349276)